libbeat/common/math.go

```diff
@@ -0,0 +1,23 @@
+package common
+
+import "math"
+
+// maxDecimalPlaces is the maximum number of decimal places that the Round
+// function return.
+var maxDecimalPlaces = 4
+
+// Round rounds the given float64 value and ensures that it has a maximum of
+// four decimal places.
+func Round(val float64) (newVal float64) {
+	var round float64
+	pow := math.Pow(10, float64(maxDecimalPlaces))
+	digit := pow * val
+	_, div := math.Modf(digit)
+	if div >= 0.5 {
+		round = math.Ceil(digit)
+	} else {
+		round = math.Floor(digit)
+	}
+	newVal = round / pow
+	return
+}
```

Comment:
🐼  I wish it was in the [standard library](https://github.com/golang/go/issues/20100), I think we should make this method takes an argument for the precision and create a method in the metrics that use the `maxDecimalPlaces`, since it's in the common file I don't think we should make arbitrary decision on the precision. 

But It can be up to discussion

Reply:
I implemented it